packages/webapp/src/webview/i18n/i18n.json

```diff
@@ -38,5 +38,5 @@
     "COPIED_TO_CLIPBOARD_DESC": "Copy this link to share via email or messages. When pasted into the search input field of the Guided Answers extension, it will navigate you straight to this node.",
     "VIEW_ON_WEBSITE": "View on the GA website",
     "COPY_WITH_INSTRUCTIONS": "Copy link with instructions",
-    "COPY_WITH_INSTRUCTIONS_TEXT": "Please follow the steps mentioned in Guided Answers extension by SAP to resolve your issue:\na. Launch Guided Answers via the command `SAP: Open Guided Answers`\nb. Paste the following guide shortlink into the search input field and hit Enter: {EXTENSION_LINK}\nc. Perform the steps as mentioned in the guide\nLearn more about the Guided Answers extension here: https://github.com/SAP/guided-answers-extension"
+    "COPY_WITH_INSTRUCTIONS_TEXT": "Please follow the steps mentioned in Guided Answers extension by SAP to resolve your issue:\na. Launch Guided Answers via the command \"SAP: Open Guided Answers\"\nb. Paste the following guide shortlink into the search input field: \"{EXTENSION_LINK}\"\nc. Perform the steps as mentioned in the guide\nLearn more about the Guided Answers extension here: https://github.com/SAP/guided-answers-extension"
```

Comment:
@oliversberzs, if you use `{{-EXTENSION_LINK}}`or perhaps better `{{-extensionLink}}` as placeholder you can call later
```typescript
i18next.t('COPY_WITH_INSTRUCTIONS_TEXT', {extensionLink: 'vscode://...'})
```
This way you don't need to manually replace, the `i18next` framework will take care of replacement.

@theStiggySpd, when using local VSCode installation (not BAS), URIs like `vscode://saposs.sap-guided-answers-extension#/tree/{TREE_ID}/actions/NODE_ID1:NODE_IDn` are directly executable URIs. Do we want to reflect this fact in the instructions?

Reply:
@Klaus-Keller Provided new text with explicit option for VSCODE and SBAS